myaws/ec2ri_ls.go

```diff
@@ -0,0 +1,95 @@
+package myaws
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	"strings"
```

Comment:
Please sort import packages with `goimports` rule. Standard libraries should be on top, third party's be later.